### ISSUE TITLE
New and modified entries for winapp2.ini (W letter)

### ISSUE THREAD

**Issue by aquila-XP** [MoscaDotTo/Winapp2#799]
`[Warhammer *]`
`Section=Games`
`DetectFile=%AppData%\The Creative Assembly\Warhammer*`
`FileKey1=%AppData%\The Creative Assembly\Launcher|*.log;*_logs.txt`
`FileKey2=%AppData%\The Creative Assembly\Launcher\electron|*.old;LOG|RECURSE`
`FileKey3=%AppData%\The Creative Assembly\Launcher\electron\Cache|*.*`
`FileKey4=%AppData%\The Creative Assembly\Launcher\electron\Service Worker\CacheStorage|*.*|RECURSE`
`FileKey5=%AppData%\The Creative Assembly\Launcher\electron\Service Worker\ScriptCache|*.*|RECURSE`
`FileKey6=%AppData%\The Creative Assembly\Warhammer*\*_cache|*.*`
`FileKey7=%AppData%\The Creative Assembly\Warhammer*\Logs|*.*`
`FileKey8=%AppData%\The Creative Assembly\Warhammer*\Temp|*.temp`

`[Waves Audio *]`
`LangSecRef=3023`
`DetectFile=%AppData%\Waves Audio`
`FileKey1=%AppData%\Waves Audio\Caches|*.*|RECURSE`
`FileKey2=%AppData%\Waves Audio\Waves Central\logs|*.*|RECURSE`
`FileKey3=%LocalAppData%\Waves Audio\logs|*.log*`
`FileKey4=%LocalAppData%\Waves Audio\Waves Central\Cache\QtWebEngine\Default\Cache|*.*`

`[WebDiscover Browser *]`
`LangSecRef=3022`
`DetectFile=%LocalAppData%\WebDiscoverBrowser`
`FileKey1=%LocalAppData%\WebDiscoverBrowser|CrashpadMetrics*.pma`
`FileKey2=%LocalAppData%\WebDiscoverBrowser\User Data\*|*.old;LOG|RECURSE`
`FileKey3=%LocalAppData%\WebDiscoverBrowser\User Data\*|Network Persistent State;Visited Links`
`FileKey4=%LocalAppData%\WebDiscoverBrowser\User Data\*\data_reduction_proxy_leveldb|*.*|RECURSE`

`[WebP Codec *]`
`LangSecRef=3023`
`DetectFile=%ProgramFiles%\WebP Codec\WebpWICCodec.dll`
`FileKey1=%ProgramFiles%\WebP Codec\Example Files|*.*|REMOVESELF`

`[WhatsApp *]`
...
`FileKey2=%AppData%\WhatsApp\*Cache|*.*|RECURSE`
`FileKey7=%AppData%\WhatsApp|*.old;LOG|RECURSE`

`[Whoer VPN *]`
`LangSecRef=3022`
`DetectFile=%ProgramFiles%\Whoer`
`FileKey1=%ProgramFiles%\Whoer|log.txt`

`[WhySoSlow *]`
`LangSecRef=3024`
`Detect=HKCU\Software\Resplendence Sp\WhySoSlow`
`FileKey1=%LocalAppData%\Resplendence|WhySoSlowLog.txt`

`[Win Movie Maker *]`
`LangSecRef=3023`
`DetectFile=%AppData%\VideoWinSoft`
`FileKey1=%AppData%\VideoWinSoft|format_his.db`
`FileKey2=%AppData%\VideoWinSoft\logs|*.*|RECURSE`

`[WinAce *]`
`LangSecRef=3024`
`DetectFile=%ProgramFiles%\WinAce\winace.exe`
`FileKey1=%ProgramFiles%\WinAce|*.cnt;*.log`

`[Winamp *]`
...
`FileKey3=%AppData%\Winamp\Plugins\ml|ml_pmp_device_*.ini`
`FileKey4=%ProgramFiles%\Winamp\Plugins\ml|ml_pmp_device_*.ini`

`[Windows Installer *]`
...
`FileKey5=%WinDir%\Installer\Config.Msi|*.*|REMOVESELF`

`[Windows Logs *]`
...
`FileKey2=%CommonAppData%\Microsoft\DiagnosticLogCSP|*.*|RECURSE`
`FileKey40=%CommonAppData%\Microsoft\Diagnosis\ETLLogs|*.*|RECURSE`
`FileKey41=%CommonAppData%\Microsoft\Windows\wfp|*.etl`
`FileKey42=%LocalAppData%\Microsoft\Dialer|*.log.txt`
`FileKey43=%LocalAppData%\Microsoft\msipc\Logs|*.*`
`FileKey44=%WinDir%\System32\CatRoot_bak|*.*|REMOVESELF`
`FileKey45=%WinDir%\System32\LogFiles\WUDF|*.*`

`[Windows Media Center *]`
...
`FileKey3=%LocalAppData%\Microsoft\eHome\Art Cache|*.*|RECURSE`

`[Windows Media Player *]`
...
`FileKey1=%LocalAppData%\Microsoft\Media Player\Cache Copertina\LocalMLS|*.*|RECURSE`
`RegKey2=HKCU\Software\Microsoft\MediaPlayer\Preferences|ObfuscatedSyncPlaylistsPath`
`RegKey3=HKCU\Software\Microsoft\MediaPlayer\Services\Bing|CachedIconPath`
`RegKey4=HKCU\Software\Microsoft\MediaPlayer\Services\Bing|CachedLargeLogoPath`
`RegKey5=HKCU\Software\Microsoft\MediaPlayer\Setup\FileMoveCache\Source`
`RegKey6=HKU\S-1-5-18\Software\Microsoft\MediaPlayer\Preferences|ObfuscatedSyncPlaylistsPath`

`[Windows Metafile Viewer *]`
`LangSecRef=3023`
`Detect=HKCU\Software\Metafile Viewer\WMF Viewer`
`RegKey1=HKCU\Software\Metafile Viewer\WMF Viewer\Recent File List`

`[Windows PowerShell *]`
...
`FileKey2=%LocalAppData%\Microsoft\Windows\PowerShell|ModuleAnalysisCache*`

`[Windscribe VPN *]`
`LangSecRef=3022`
`DetectFile=%ProgramFiles%\Windscribe`
`FileKey1=%LocalAppData%\Windscribe\Windscribe|*.log;*log.txt`
`FileKey2=%ProgramFiles%\Windscribe|*.log`
`FileKey3=%ProgramFiles%\Windscribe\log|*.*|RECURSE`

`[WinExt Registry Fixer *]`
`LangSecRef=3024`
`DetectFile=%AppData%\TSS\WERF`
`FileKey1=%AppData%\TSS\WERF|REG_POINT_*.reg`

`[WinFast *]`
`LangSecRef=3023`
`Detect=HKCU\Software\Leadtek\Video\WFFM`
`FileKey1=%WinDir%\System32\WinFast\Multimedia\DirectBurn|*.log`
`RegKey1=HKCU\Software\Leadtek\Video\WFFM|Last Name`

`[WinThruster *]`
`LangSecRef=3024`
`Detect=HKCU\Software\WinThruster`
`DetectFile=%AppData%\WinThruster`
`FileKey1=%AppData%\WinThruster\Backup|*.*`
`FileKey2=%AppData%\WinThruster\Log|*.*`
`FileKey3=%AppData%\WinThruster\Undo|*.*`
`RegKey1=HKCU\Software\WinThruster|InstallerName`
`RegKey2=HKCU\Software\WinThruster|LastScanDate`
`RegKey3=HKCU\Software\WinThruster|SizeToClean`

`[WinTools.net *]`
`LangSecRef=3024`
`DetectFile1=%ProgramFiles%\Godlike Developers\WinTools.net*`
`DetectFile2=%ProgramFiles%\WinTools Software\WinTools.net*`
`FileKey1=%ProgramFiles%\Godlike Developers\WinTools.net*|*.txt;*.url`
`FileKey2=%ProgramFiles%\Godlike Developers\WinTools.net*\Undo|*.reg|RECURSE`
`FileKey3=%ProgramFiles%\WinTools Software\WinTools.net*|*.txt`
`FileKey4=%ProgramFiles%\WinTools Software\WinTools.net*\Undo|*.reg|RECURSE`

`[WinX YouTube Downloader *]`
...
`DetectFile1=%AppData%\Digiarty\WinX YouTube Downloader`
`DetectFile2=%AppData%\WinX YouTube Downloader`
`FileKey2=%AppData%\WinX YouTube Downloader|log.txt`

Note: Please remove the old detection lines.

`[WinX DVD Ripper Platinum *]`
`LangSecRef=3023`
`DetectFile=%AppData%\Digiarty\WinX DVD Ripper Platinum`
`FileKey1=%AppData%\Digiarty\WinX DVD Ripper Platinum|*.log;*.tmp`

`[WinZip *]`
`LangSecRef=3024`
`Detect=HKCU\Software\Nico Mak Computing\WinZip`
`DetectFile=%LocalAppData%\Nico Mak Computing\WinZip Update Notifier`
`FileKey1=%LocalAppData%\Nico Mak Computing\WinZip Update Notifier\log|log-*.txt`
`RegKey1=HKCU\Software\Nico Mak Computing\WinZip\Mru`

Note: Please remove `[WinZip MRU *]`.

`[Wireshark *]`
`LangSecRef=3024`
`DetectFile=%ProgramFiles%\Wireshark`
`FileKey1=%ProgramFiles%\Wireshark|*Setup*.exe;npcap-*.exe`

`[WiseVector StopX *]`
`LangSecRef=3024`
`DetectFile=%ProgramFiles%\WiseVector\WiseVector.exe`
`FileKey1=%ProgramFiles%\WiseVector\log|*.log`

`[Wizards of the Coast *]`
`Section=Games`
`DetectFile=%LocalLowAppData%\Wizards of the Coast`
`FileKey1=%LocalLowAppData%\Wizards of the Coast\mtga|*_log.txt`

`[Wondershare Video Converter *]`
`LangSecRef=3023`
`DetectFile=%ProgramFiles%\Wondershare\Video Converter*`
`FileKey1=%CommonAppData%\Wondershare Video Converter*|*.dat.bak`
`FileKey2=%CommonAppData%\Wondershare Video Converter*\TempSiteIconDir|*.*`
`FileKey3=%CommonAppData%\Wondershare Video Converter*\TempThumbDir|*.*`
`FileKey4=%CommonProgramFiles%\Wondershare\Wondershare Helper Compact\log|*.*|RECURSE`
`FileKey5=%CommonProgramFiles%\Wondershare\Wondershare Helper Compact\Temp|video*.exe`
`FileKey6=%ProgramFiles%\Wondershare\Video Converter*\Log|*.*|RECURSE`
`FileKey7=%ProgramFiles%\Wondershare\Video Converter*\Output|Thumbnail.dat`
`FileKey8=%ProgramFiles%\Wondershare Video Converter*\TempThumbDir|*.*|RECURSE`
`FileKey9=%ProgramFiles%\Wondershare\Video Converter*\Transfer\MultimediaLibs\Log|*.*`

**Comment by APMichael** [comment]
`[Wondershare Video Converter *]`
Almost everything already covered by CCleaner.

**Comment by aquila-XP** [comment]
You are right. Those lines that I submitted for `[Wondershare Video Converter *]` entry are already covered by CCleaner with `[Wondershare Video Converter Pro]` and `[Wondershare Video Converter Ultimate]` entries.
The reason I submitted those lines is because there are different versions of "Wondershare Video Converter" program: Wondershare Video Converter Free, Wondershare Video Converter Pro and Wondershare Video Converter Ultimate.
